src/keys.c

```diff
@@ -371,7 +373,10 @@ load_keys(const char* jwkdir)
             filepath[sizeof(filepath) - 1] = '\0';
             json_auto_t* json = json_load_file(filepath, 0, NULL);
             if (!json) {
-                fprintf(stderr, "Invalid JSON file (%s); skipping\n", filepath);
+                enum json_error_code code = json_error_code(&error);
```

Comment:
This `json_error_code()` was added in jansson 2.11, which is why it's failing in the CI for Centos 7, as it has 2.10. On the other hand, I am not sure it adds much vaulable info in this case. Perhaps you could drop `code` here and in the `fprintf` below, and keep only the text, line, column and position?

Reply:
Will drop.